### DISCUSSION TITLE
loh3.yaml@1: Could not find model for point [ 0 0 0 ] in group 0

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#646] [Q&A] [answered]
I modified the input .msh file on the basis of the Example of loh3 model, which is no longer generated by the .geo file, but directly imported a .msh file as shown in the figure below. 

![image](https://user-images.githubusercontent.com/104721358/186166985-bf0bf4c5-cb3f-4739-b85d-06bd17f71d8a.png)

The mesh file I used does not contain point(0, 0, 0), so I encountered the following error, 
![image](https://user-images.githubusercontent.com/104721358/186159164-064d8a80-af30-4cc1-9aaa-7a6933a8062f.png)

I have changed the model， why this problem occurs？ can this problem be solved by modifying one of the files below?

![image](https://user-images.githubusercontent.com/104721358/186168776-20df012a-7030-4ed4-880f-62918bf01197.png)

**Comment by sebwolf-de** [comment, marked as answer]
Hi, I tried the LOH3 model by copying, `loh.puml.h5` and `source.nrf` from the LOH1 folder to the LOH3 folder. For me the scenario works then.
> I have changed the model， why this problem occurs?

I can only help you, when you tell me, what you have changed. You should not change the `.msh` file. It is only an intermediate representation of the mesh, generated by gmsh. To change the geometry, you should change the `.geo` file and then run `generating_the_mesh.sh` again.

**Reply by smallriver666** [reply]
Thanks for the reply, that's a model I made myself, I know why it goes wrong
